### PR TITLE
MOTECH-2865: Travis integration tests on mysql and postgresql

### PR DESCRIPTION
**Pull request by mkruszynski**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - sudo service tomcat7 stop
+  - echo 'CATALINA_OPTS="-Xms512m -Xmx512m"' | sudo tee --append /usr/share/tomcat7/bin/setenv.sh
+  - sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/ /usr/share/tomcat7/ /var/log/tomcat7/
+
+before_script:
+  - sudo chmod +x ./travis_before_build.sh
+  - ./travis_before_build.sh
+
+script:
+  - mvn clean install -PIT -q
```
Comment (sebbrudzinski) [inline .travis.yml]: Despite this, I can see that the Travis console outputs the following:
`mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V`
Can you verify where is that coming from?

Comment (mkruszynski) [inline .travis.yml]: This is a default command in "install" phase of build. If we want to change this, we just should add e.g. "install: mvn clean install" to this file. 
Here it is in Travis documentation:
https://docs.travis-ci.com/user/languages/java/#Dependency-Management

Comment (sebbrudzinski) [inline .travis.yml]: So do we separately run install and then tests now?

Comment (mkruszynski) [inline .travis.yml]: Yes, I'm checking now a configuration with tests only. If they will succeed, I'll push a new configuration file.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y activemq -qq
+  - sudo ln -s /etc/activemq/instances-available/main /etc/activemq/instances-enabled/main
+  - sudo sed -e 's/<broker /<broker schedulerSupport="true" /' -i /etc/activemq/instances-enabled/main/activemq.xml
+  - sudo service activemq restart
+  - sudo apt-get install -y tomcat7
+  - sudo service tomcat7 stop
+  - echo 'CATALINA_OPTS="-Xms512m -Xmx512m"' | sudo tee --append /usr/share/tomcat7/bin/setenv.sh
+  - sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/ /usr/share/tomcat7/ /var/log/tomcat7/
```
Comment (sebbrudzinski) [inline .travis.yml]: Can we move all of this to a script as well? Also, let's start naming them adequately to what they do (eg. set_up_databases.sh, install_dependencies.sh etc.)

Comment (mkruszynski) [inline .travis.yml]: Of course, I'll handle this.

**Comment by sebbrudzinski** [comment]
@mkruszynski  Can we close it now?

**Comment by mkruszynski** [comment]
@sebbrudzinski Yes, I think this PR isn't more necessary. We have similar configuration in PR for MOTECH-2895.

**Pull request closed without merging**